### DISCUSSION TITLE
Xarray website UI design

### DISCUSSION THREAD

**Discussion by andersy005** [pydata/xarray#5621] [Ideas]
@pydata/xarray, 

With @jhamman's input, I have been working on a UI prototype for an Xarray website. This new website is one of the milestones/deliverables for the CZI-EOSS2 Proposal. The image and link to the **live** [figma](https://www.figma.com/) project below are just an initial prototype so we can get feedback on things like _style_, _contents_, etc. Once folks have had the opportunity to provide feedback, I will start working on the actual implementation. 

- Figma Project URL: https://www.figma.com/file/olCObSTsPg5cMmSpBEramU/landing-site?node-id=2%3A2

![xarray-landing-page](https://user-images.githubusercontent.com/13301940/126217608-1cf77f71-057f-4c4d-81f5-b4820dcd8513.png)

**Comment by mathause** [comment]
Looks good! For me the fonts are too playful (mostly the monospace font). And I don't especially like the `apply_ufunc` example because that's not an example for which it is necessary to use `apply_ufunc`.

(There are some typos
 - arbitrary
 - of _a_ Python dictionary
 - `import xarray` (needs `as xr`)
 - Running the example code will require pooch and netCDF4 or scipy so the installation instructions are not enough
 - )

**Reply by shoyer** [reply]
This is a wonderful start, indeed!

I agree that code example could be simplified -- apply_ufunc is really advanced feature, probably best saved for later!

Let's also add Nvidia to the sponsors list, they funded a bunch of development this past year via Numfocus.

**Reply by keewis** [reply]
I would probably also either remove the in-text enumeration of projects or add a few more example projects (and point to the "ecosystem" page for the extensive list)

**Comment by dcherian** [comment]
We could add similar rows of logos under "Wraps array types" and "Supports backend datastores" (or similar) but maybe that would be too much.

I agree that is a great improvement!

**Reply by benbovy** [reply]
Agreed, upstream ecosystem integration seems equally important than downstream libraries. Numpy, Dask, Zarr, etc. logos would be welcome in this section too I think.

Perhaps using tabs like in the "ecosystem" section of https://numpy.org/?

<img width="1212" alt="Screenshot 2021-07-20 at 14 09 27" src="https://user-images.githubusercontent.com/4160723/126321656-01e5e857-ef90-40de-bf8a-2076531633da.png">

Overall this looks already very nice!

**Comment by andersy005** [comment]
Thank you all for the feedback...I'll adjust the UI prototype accordingly

**Comment by astrojuanlu** [comment]
Hi folks, just casually landed at the new [xarray website](https://xarray.dev/) and it's _beautiful_, congrats! I'm curious, was this done by someone from the community that already had the skills, or rather an external person or team was involved? I'd love to know more in case [we in poliastro undertake a similar revamp of our website](https://github.com/poliastro/poliastro/issues/1474).

**Reply by TomNicholas** [reply]
As far as I know it was just @andersy005 and @jhamman , who were already in our community. Nice to hear you like the website so much though!

**Reply by jhamman** [reply]
@TomNicholas is right. We had a bit of prior experience building React/Next.js sites which helped. The site is deployed via Vercel and is open source: https://github.com/xarray-contrib/xarray.dev
